### ISSUE TITLE
Mise à jour description du format irve statique

### ISSUE THREAD

**Issue by stephane-pignal** [etalab/schema-irve#62]
Concerne le texte https://schema.data.gouv.fr/etalab/schema-irve-statique/2.2.0/documentation.html

Objectif : une meilleure compréhension de certains champs pour les producteurs ainsi qu'une montée en en qualité des données.

Constat : suite aux différentes entretiens Affirev, Avere, Gireve et à des retours producteurs, il est opportun d'apporter des précisions/modifications/ajouts sur la documentation du format IRVE statique.

Exemple :  champ Propriété siren_amenageur, rajouter "composée de 9 chiffres". Permet d'éviter le confusion avec le n° de SIRET (13 chiffres).

**Comment by ptitfred** [comment]
J'imagine que tu avais déjà vu ceci qui semble faire (à sa façon) ce tu suggères dans ton exemple

![image](https://github.com/etalab/schema-irve/assets/346377/b50fb188-de38-4b06-888f-86315b12196b)

S'agit-il de le reformuler pour le rendre plus explicite ?

**Comment by stephane-pignal** [comment]
Yep objectif de plus d'autonomie pour moins de support sur des erreurs de ce type

**Comment by stephane-pignal** [comment]
Je te rejoins, cela devrait largement suffire.... mais non
Ces propositions de modifs sont issues du support côté producteurs où nous avons des profils d'interlocuteurs très divers. Par ex. ce matin, une personne qui s'occupe d'administration des ventes et de constituer le fichier csv.
Ma proposition serait "Le numero SIREN (9 chiffres) de l'aménageur issue de la base SIRENE des entreprises. Vous pouvez récupérer cet identifiant sur le site annuaire-entreprises.data.gouv.fr."
